crescent/internal/meta_struct.py

```diff
@@ -27,6 +27,7 @@ class MetaStruct(Generic[T, U]):
     _app: Optional[Bot] = None
 
     app_set_hooks: List[Callable[[MetaStruct[T, U]], None]] = field(factory=list)
+    plugin_unload_hook: Callable[[MetaStruct[T, U]], None] | None = None
```

Comment:
For consistency (and for the sake of extensions) I would make this a list rather than a single hook. Every other `*hook*` so far has been a list.

Also, do we want `plugin_load_hooks`?

Reply:
First change I'll make.
There's no need for plugin load hooks. They would function exactly the same as app set hooks.